examples/usage/README.md

```diff
@@ -0,0 +1,35 @@
+# Example: Usage
+
+This primary usage example is also included in the [README.md](../../README.md) of this module. It demonstrates the virtual machine module using as many default values as possible.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- [Terraform](https://www.terraform.io/downloads.html) installed on your local machine.
+- An [Azure account](https://azure.microsoft.com/en-us/free/) with the appropriate permissions.
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed and authenticated.
+
+## Steps to Deploy
+
+1. **Initialize the Terraform Configuration**
+
+   Initialize the Terraform configuration to download the necessary providers and modules.
+
+   ```shell
+   terraform init
+   ```
+
+2. **Apply the Terraform Configuration**
+
+   Apply the configuration to create the resources defined in your Terraform files.
+
+   ```shell
+   terraform apply
+   ```
```

Comment:
This doesn't work, which is foreseeable because the examples in the example values, e.g., in the Management Group.

I see the following possibilities:

- Remove the “Steps to Deploy”
- Or make example deployable with `*_overwrite.tf` files

Reply:
I removed it